src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/physicalattributes/PhysicalAttributesDpsApiServiceTest.kt

```diff
@@ -22,13 +22,13 @@ import java.time.LocalDateTime
 import java.time.ZoneId
 
 @SpringAPIServiceTest
-@Import(PhysAttrDpsApiService::class, PrisonPersonConfiguration::class, PhysAttrDpsApiMockServer::class)
-class PhysAttrDpsApiServiceTest {
+@Import(PhysicalAttributesDpsApiService::class, PrisonPersonConfiguration::class, PhysicalAttributesDpsApiMockServer::class)
+class PhysicalAttributesDpsApiServiceTest {
   @Autowired
-  private lateinit var apiService: PhysAttrDpsApiService
+  private lateinit var apiService: PhysicalAttributesDpsApiService
 
   @Autowired
-  private lateinit var physAttrDpsApi: PhysAttrDpsApiMockServer
```

Comment:
Mostly I've expanded to use full words, and it does look better. Apart from some of the class variables where there is no ambiguity and I've been able to shorten the name, such as here.